source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -1,121 +1,155 @@
-example 18220 becomes 18220000, showing the base version ``18220`` and allowing
-for ``100`` new "mixes" or package adds within that version (increment by
-10 x 100). By doing so, swupd can still check if there is a version > 18220,
-and if so, it will automatically generate the new Manifest.MoM for the user on
-the next update operation. This means autoupdate will still work, and users
-will continue recieving security updates per normal update schedule. On top
-of this, a mixed file will exist at :file:``/usr/share/defaults/swupd/mixed``
-signifying that the system is on an augmented version.
-
-How to use it
-=============
-Using the new feature is simple. Create a new folder :file:``/usr/share/mix/``,
-which is where the local update content will go. In that folder, create an
-"rpms" directory, and simply :command:``cp`` your RPMs into it. *Note that
-the RPMs must be formatted and build for Clear.
-
-#. **Adding RPMs to local folder**
-Copy files into correct folder::
-      # sudo cp <RPMs you want to add> /usr/share/mix/rpms/
-
-   This is where the swupd-add-pkg script looks to for its workspace.
-
-#. **Add Rpms and create content**
-run::
-      # sudo swupd-add-pkg <pkg_name> <bundle_name>
-
-   This automatically creates the <bundle_name> bundle if it does not exist,
-   adds the RPM name to it, and generates all the mixer content and Manifests
-   needed.
-
-#. **Migrate to local setup**. Now that the content exists, the easiest way to have swupd
-   become aware of it and use it, is to use the new :command:``--migrate`` flag for an update.
-run::
-   # sudo swupd update --migrate
-
-   This will update your system to the mix ready state using the new Manifest.MoM.
-
-#. **Add your bundle to your system** To add your bundle, simply run::
-
-   # sudo swupd bundle-add <my new bundle>
-
-
-
-Revert back to upstream
--------------------------
-
-You may wish to go back to official upstream with no user content. This can
-easily be done by calling :command:``swupd verify --fix --picky --force -m
-<upstreamversion> -C /usr/share/clear/update-ca/Swupd_Root.pem``, which will
-verify fix the system back to the upstream version you are augmenting with
-local content. From here on regular swupd commands will only reach out to
-the upstream server, since /usr/share/mix will no longer exist.
+*************
+
+To use the integrated swupd mixer, you need a recent image of
+|CLOSIA| with the mixer bundle installed. If you do not have it already,
+add it with the :command:`swupd bundle-add` command as shown below:
+
+.. code-block:: bash
+
+   sudo swupd bundle-add mixer
+
+
+Using the enhanced swupd mixer
+******************************
+
+Perform the following steps:
+
+#. Create a folder :file:`/usr/share/mix/rpms` for the local update content
+   and copy the desired RPMs into the folder:
+
+   .. note::
+
+     The RPMs must be formatted and built for |CL|.
+
+   .. code-block:: bash
+
+      sudo cp <RPM1 RPMn> /usr/share/mix/rpms/
```

Comment:
A list of RPMs is not supported yet, please change these to the original representation.